### PR TITLE
Jim Francis added to the list of alert recipients

### PR DESCRIPTION
**Pull request by Gregg140**
Jim Francis added to limitpager and acis-P3-fluence alert lists. In addition, acis-P3-fluence also sends emails via OpsGenie.

### CODE REVIEW

File: acis-P3-fluence.pl [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+
+   $DickEEmail = "redgar\\\@cfa.harvard.edu";
+   $DickEPhone = "6178668615\\\@tmomail.net";
+
+   $RoyceBEmail = "buehler\\\@space.mit.edu";
+   $RoyceBPhone = "6178518470\\\@vtext.com";
+
+   $JackSteinerEmail = "James.Steiner\\\@cfa.harvard.edu";
+   $JackSteinerPhone = "6176809306\\\@vtext.com";
+
```
Comment (jazan12) [inline acis-P3-fluence.pl:741]: You don't have Jim in here!

**Review by jazan12 — COMMENTED** [review]
I flagged a spot I think Jim needs to be added still.

**Comment by jzuhone** [comment]
Looks ok except for the issue that Jack noted.

File: acis-P3-fluence.pl (hunk 1 above)
Comment (Gregg140) [reply acis-P3-fluence.pl:741]: Ok Jim was added and also I modified the acis-p3-fluence.pl program to utilize a -t command line switch for test purposes.
This causes the program to use very low values for the limits used to determine whether or not to send emails.

**Review by jazan12 — APPROVED** [review]
Bueno!

**Pull request merged**